third_party/openthread/src/ncp/spinel.h

```diff
@@ -979,6 +979,9 @@ typedef enum
      */
     SPINEL_PROP_IPV6_ICMP_PING_OFFLOAD  = SPINEL_PROP_IPV6__BEGIN + 5, ///< [b]
 
+    SPINEL_PROP_IPV6_MULTICAST_ADDRESS_TABLE
```

Comment:
I think realizing this probably require changes beyond the ncp/spinel and most likely into OT core.